### DISCUSSION TITLE
eMacs through WebSSH

### DISCUSSION THREAD

**Discussion by d-rideout** [isontheline/pro.webssh.net#803] [Q&A]
I would like to use eMacs through WebSSH, but find it unusable.  How do I enter the Meta key?   And Ctrl-_??
I am using the standard Mac external keyboard that came with the iPad. Neither option nor cmd nor Esc seem to work for Meta.  Can there be a HOWTO posted somewhere to explain how to map keys, which is understandable by 'iPad newbies'?  I find all the conflicting instructions in various places extremely confusing.

(And why does my iPad automatically capitalize the 'm' in eMacs??  Because the 'm' is always capitalized when followed by 'ac'???  (just showing some of my general frustration, to illustrate what I mean by 'iPad newbie'...))

**Comment by isontheline** [comment]
Hello @d-rideout David 👋

Sorry to hear that you are encountering frustration while using WebSSH.

Have you tried this howto?

https://webssh.net/documentation/help/howtos/SSH/use-meta-alt-option-key/

Perhaps you should use a better built product such as Termius which have a greater support than WebSSH.

Have a nice day ☀️

**Reply by d-rideout** [reply]
Thank you!

I did try that howto, but nothing mentioned there works for me.  Perhaps I am hoping for something which goes into more depth, like the Linux HOWTOs of old.  So that I can have a chance of diagnosing why it is not working for me.

Thank you for the Termius suggestion.  The option key does act as Meta in emacs in Termius.  Though the Ctrl-_ (undo) does not work there either.  Probably something is going wrong because I also need to hold down the shift key to access the underscore character, but then the Ctrl-Shift combination is being captured by something else before it reaches emacs.
